src/lib.rs

```diff
@@ -121,6 +161,24 @@ impl Span {
     pub fn def_site() -> Span {
         Span(imp::Span::def_site())
     }
+
+    pub fn source_file(&self) -> SourceFile {
+        SourceFile(self.0.source_file())
+    }
+
+    pub fn start(&self) -> LineColumn {
+        // XXX(nika): We can't easily wrap LineColumn right now
+        self.0.start()
+    }
+
+    pub fn end(&self) -> LineColumn {
+        // XXX(nika): We can't easily wrap LineColumn right now
```

Comment:
As of https://github.com/rust-lang/rust/pull/46690, the fields have been made public in the proc-macro implementation, so we can probably do a wrap of this type now.